package com.github.savitoh.bytebank.auth.exception

class FalhaAutenticacaoException(
    mesage: String = "Falha na autenticação"
) : Exception(mesage)